packages/vscode-extension/package.json

```diff
@@ -543,12 +543,12 @@
       },
       {
         "command": "fx-extension.preview",
-        "title": "Preview",
+        "title": "Preview Teams App",
```

Comment:
As now we also have ability to extend Teams app to outlook and office, should we use Preview App instead of Preview Teams App to prevent further change?

Reply:
Got it, changed it to 'Preview App'